pkgs/development/python-modules/cassandra-driver/default.nix

```diff
@@ -92,6 +98,8 @@ buildPythonPackage rec {
   disabledTestPaths = [
     # requires puresasl
     "tests/unit/advanced/test_auth.py"
+    # Uses asyncore, which is deprecated in python 3.12+
+    "tests/unit/io/test_asyncorereactor.py"
```

Comment:
Can you add a comment to drop this on the next release? https://github.com/datastax/python-driver/commit/9952e2ab22c7e034b96cc89330791d73c221546b skips it natively. (We could alternatively patch with that commit, if it applies; it also moves to pytest fixtures, which might be more reliable.)

Reply:
Added this patch and a previous one that drops mock as a dependency(since it's needed for the suggested patch to apply)